server/models/Card.js

```diff
@@ -0,0 +1,57 @@
+const mongoose = require("mongoose");
+
+const cardSchema = new mongoose.Schema({
+  _id: {
+    type: mongoose.Types.ObjectId,
+    required: true,
+  },
```

Comment:
As mentioned in Column schema, `id` will be added on by mongoose automatically which means this is redundant.